src/qt/sendcoinsdialog.cpp

```diff
@@ -186,6 +187,13 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         // set default rbf checkbox state
         ui->optInRBF->setCheckState(Qt::Checked);
 
+        if (model->privateKeysDisabled()) {
+            ui->sendButton->setText(tr("Create Unsigned"));
+            ui->sendButton->setToolTip(tr("Creates a Partially Signed Bitcoin Transaction (PSBT) for use with e.g. an offline Bitcoin Core wallet, or a PSBT-compatible hardware wallet."));
```

Comment:
Do we want to imply the Bitcoin Core GUI can sign these yet?

Reply:
In reference to "offline Bitcoin Core wallet"? It doesn't say GUI :-)

Reply:
Hence "imply".

Reply:
I only expect people to use this feature (watch only wallets in general) based on some tutorial, so I don't think this needs to be perfect. Hopefully we can add signing PSBT from the GUI soon too.